pilot/pkg/model/context.go

```diff
@@ -717,17 +715,17 @@ func ParsePort(addr string) int {
 	return port
 }
 
-// parseIPAddresses extracts IPs from a string
-func parseIPAddresses(ipAddresses []string) ([]string, error) {
+// hasValidIPAddresses returns true of passes in slice has all valid ips, otherwise returns false.
```

Comment:
// hasValidIPAddresses returns true if the input ips are all valid, otherwise returns false. ？